lib/shared/catalina-sudo/sudo-askpass.osascript.js

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env osascript -l JavaScript
+
+/* eslint-disable */
+
+ObjC.import('stdlib')
+
+const app = Application.currentApplication()
+app.includeStandardAdditions = true
+
+const result = app.displayDialog('Please enter your sudo password.', {
```

Comment:
I don't have a Mac myself, but will the average Mac user know what "your sudo password" means?
I wonder if it might make sense to copy the text used in the screenshots on https://github.com/jorangreef/sudo-prompt ?
:man_shrugging: